src/App.tsx

```diff
@@ -1,27 +1,64 @@
-import React from 'react';
+import React, { useState } from 'react';
+import 'bulma';
 import './App.scss';
 import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [itemsGoods, setItemsGoods]: any[] = useState([]);
```

Comment:
You can set 
`'useState<Good[]>([]);'` 
it should remove the warning
`'any[]'` is not very correct in this case